roles/tpa_single_node/templates/systemd/systemd-cm2.j2

```diff
@@ -0,0 +1,18 @@
+[Unit]
```

Comment:
Change the name with something meangiful that describe the purpose, like systemd-twocm.j2 or systemd-two-cm.j2 otherwise seems only a second version of the systemd-cm.j2

Reply:
Done